duckduckgo/iOS
# prepare_release workflow improvements

Task/Issue URL: https://app.asana.com/0/414235014887631/1203800022093370/f

**Description**:
- support for release notes,
- support for metadata updates,
- validation of metadata

**Steps to test this PR**:
1. Run the flow and see if release notes are properly commited.
2. Run the flow and see if metadata is properly updated (either zipped one or not).

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
